### PR TITLE
Josh Salyer

### PR DESCRIPTION
**Pull request by jsalyer23**
I don't understand why .toUppercase() doesn't pass the tests. Is it because it doesn't actually change the String? The documentation i've been looking at for JavaScript Arrays has examples with HTML script which is confusing me a little since there isn't any in the Cloud9 assignment. Where might I find some clearer resources besides the Codecademy tutorials?

### CODE REVIEW

**Comment by jsalyer23** [comment]
I figured out what I was doing wrong before so just ignore my previous comment. It helps when I use .toUpperCase() instead of .toUppercase(). Oops!

**Comment by halfghaninne** [comment]
Eep! Yes that will help! JavaScript is very case sensitive. Thanks for your patience in us getting to your work and kudos for working through that on your own in the meantime.

File: lib/3_add_up_numbers.js [hunk 1]
```diff
@@ -6,6 +6,9 @@
 
 module.exports = {
   add_up_numbers: function(arr){
-    
+   var total = arr.reduce(function(a, b){
```
Comment (halfghaninne) [inline lib/3_add_up_numbers.js:9]: Nice!

File: lib/4_filter_numbers.js [hunk 2]
```diff
@@ -6,6 +6,8 @@
 
 module.exports = {
   filter_numbers: function(arr){
-    // Your code goes here.
+  var lessThan10 = arr.filter(function (x){
```
Comment (halfghaninne) [inline lib/4_filter_numbers.js:9]: Within the outermost function, spacing your work like so makes it easier to parse:

``` javascript
var lessThanTen = arr.filter(function(x) {
  return x < 10;
});
return lessThanTen;
```

**Comment by halfghaninne** [comment]
👍  Nice work!

**Pull request closed without merging**